hipe4ml/plot_utils.py

```diff
@@ -1,45 +1,61 @@
 """ Module containing the plot utils. Each function returns a matplotlib object
     """
-from inspect import signature
-import numpy as np
-from mpl_toolkits.axes_grid1 import ImageGrid
 import matplotlib.pyplot as plt
-from pandas.core.index import Index
+import numpy as np
 import pandas as pd
+from pandas.core.index import Index
+import shap
+from mpl_toolkits.axes_grid1 import ImageGrid
 from sklearn.metrics import (auc, average_precision_score,
                              precision_recall_curve, roc_curve)
 
-import shap
-
 
 # plot the BDT score distribution in the train and in the test set for both signal and background
 def plot_output_train_test(
-        clf, x_train, y_train, x_test, y_test, model='xgb', features=None, bins=80):
-    '''
-    Plot the BDT output for the signal and bckground distributions
+        clf, data, model='xgb', features=None, bins=80, raw=True, **kwds):
+    """
+    Plot the BDT output for the signal and background distributions
     both for training and test set.
 
     Input
     ----------------------------------------
-    x_train: training set
-    y_train: training label
-    x_test: test set
-    y_test: test label
+    x_train: Pandas dataframe
+    Training set
+
+    y_train: list or array
+    Training set labels
+
+    x_test: Pandas dataframe
+    Test set
+
+    y_test: list or np.array
+    Test set labels
+
```

Comment:
This parameters are no more present, right? Moreover, the parameters ```clf```, ```bins``` and  ```raw``` are not described.